### ISSUE TITLE
Cannot find module 'eslint-plugin-jsx-a11y'

### ISSUE THREAD

**Issue by marioy47** [pnpm/pnpm#2680]
### Code to reproduce the issue:
I'm using vim with Conquer of Copletition to develop React application. And the ESLint tools is not finding the `eslint-plugin-jsx-a11y`

Not sure but this could be a eslint-plugin error, a CoC error. But using `npm` or `yarn` does not present this error.

### Additional information:

 - `node -v` prints: v14.4.0
 - Windows, OS X, or Linux?: Mac OS Catalina 10.15.5 (19F101)

```
[coc.nvim] Error on willSaveWaitUntil: Error: Request textDocument/willSaveWaitU
ntil failed with message: Failed to load plugin 'jsx-a11y' declared in '.eslintr
c.json » plugin:@wordpress/recommended » /Users/mario/Projects/wp-devenv/wp/wp-c
ontent/plugins/ihs-appointments/node_modules/.pnpm/@wordpress/eslint-plugin@7.1.
0_eslint@7.4.0/node_modules/@wordpress/eslint-plugin/configs/recommended-with-fo
rmatting.js » /Users/mario/Projects/wp-devenv/wp/wp-content/plugins/ihs-appointm
ents/node_modules/.pnpm/@wordpress/eslint-plugin@7.1.0_eslint@7.4.0/node_modules
/@wordpress/eslint-plugin/configs/jsx-a11y.js': Cannot find module 'eslint-plugi
n-jsx-a11y'
Require stack:
- /Users/mario/Projects/wp-devenv/wp/wp-content/plugins/ihs-appointments/__place
holder__.js
Referenced from: /Users/mario/Projects/wp-devenv/wp/wp-content/plugins/ihs-appoi
ntments/node_modules/.pnpm/@wordpress/eslint-plugin@7.1.0_eslint@7.4.0/node_modu
les/@wordpress/eslint-plugin/configs/jsx-a11y.js
```

![Screenshot](https://user-images.githubusercontent.com/3419025/86993588-fdae3200-c169-11ea-8cc8-963d9330c95b.png)

**Comment by zkochan** [comment]
Try to create a `.npmrc` with the following content in the root of your project:

```
public-hoist-patter[]=eslint-plugin-*
```

Then run `pnpm install`.

**Comment by marioy47** [comment]
Yes, that did the trick.

Thanks for such a great tool

**Comment by zkochan** [comment]
Ok, we'll add it to the default list of patterns

https://github.com/pnpm/pnpm/blob/2ab9b9e17f2da3069d9ee9f6cf6b50b3713aa6a4/packages/config/src/index.ts#L150
